### ISSUE TITLE
"Prepare release" template "Finish App translations" change

### ISSUE THREAD

**Issue by pljones** [jamulussoftware/jamulus#3361]
Currently, "Finish App translations" reads"
> - [x] Finish App translations
>   - [x] Review translation PRs according to [release process checklist](https://jamulus.io/contribute/Release-Process#3-update-the-ts-files-returned-by-translators)
>   - [x] Wait for all PRs to be merged (missing translations will revert to English automatically).
>   - [ ] Check for conflicting accelerator keys (see tools/checkkeys.pl)
>   - [ ] [Check for broken links](https://github.com/jamulussoftware/jamuluswebsite/blob/release/contribute/en/Release-Process.md#website-check-links)

"Check for broken links" is a website check -- so it needs removing from the "Finish App translations".

_Originally posted by @pljones in https://github.com/jamulussoftware/jamulus/issues/3255#issuecomment-2321897409_

**Comment by ann0see** [comment]
Yes. Even though we link to the website from within the app, I don't think it's the case in many places
